### PR TITLE
update to description of files generated from examples, which are no …

### PR DESCRIPTION
**Pull request by londoncalling**
…longer owned by root w/new release

Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

**Comment by londoncalling** [comment]
@dnephin @shin- @moxiegirl @thaJeztah 

I went through the Getting Started, along w/django, rails, and wordpress examples. (All worked, except wordpress which gave me an EOF error I can show you in Slack.) 

Docs changes in this PR:

-  removed tutorials (now a private repo) path from Dockerfile so that the docs build will run (@moxiegirl: I assume this is okay to check this in as modified?)

- update the listings of generated files. They used to show up as owned by root, and the user had to change ownership/permissions,  but it looks like with this release that is no longer necessary.

- Added pix of web sites you get at the end of the example walk-throughs

- Copy-edits on the overview

**Comment by thaJeztah** [comment]
@londoncalling noticed your screenshots contain the OS X drop-shadow; you can disable the drop-shadow in your screenshots using;

    defaults write com.apple.screencapture disable-shadow -bool TRUE; killall SystemUIServer

(only need to do that once). More info can be found here [How to Eliminate Drop Shadows in OS X Screenshots](https://tidbits.com/article/15616)

**Comment by thaJeztah** [comment]
w.r.t. the changes in permissions, I'm not sure this is correct; I think this is only the case if you're using boot2docker / VirtualBox. If you're running on Linux, the files are still created as root;

```
drwxr-xr-x 2 root       root   4096 Feb 15 05:35 composeexample
-rw-r--r-- 1 sebastiaan docker  160 Feb 15 05:29 docker-compose.yml
-rw-r--r-- 1 sebastiaan docker  147 Feb 15 05:28 Dockerfile
-rwxr-xr-x 1 root       root    257 Feb 15 05:35 manage.py
-rw-r--r-- 1 sebastiaan docker   17 Feb 15 05:28 requirements.txt
```

**Comment by londoncalling** [comment]
Thanks Seb, I'll back out that change and add an explanation that this only
applies if you are on Linux.

**Comment by londoncalling** [comment]
@thaJeztah @moxiegirl @dnephin @shin- 

Please take a look now, w/Seb's comments addressed. Let me know if these changes LGTYou and whether we can merge.  (I'd also like to get the Wordpress error working and figure out what the problem was so we can make any notes necessary in the docs.)

**Comment by dnephin** [comment]
Needs a rebase, otherwise LGTM

**Comment by thaJeztah** [comment]
LGTM as well (unless you want to wrap those lines to 80-chars)

**Comment by londoncalling** [comment]
@moxiegirl @thaJeztah @dnephin rebase and line wrap fixes done. If build completes successfully, can someone merge this for me? Thanks!

**Comment by thaJeztah** [comment]
sorry, don't have write access to this repo

File: docs/overview.md [hunk 1]
```diff
@@ -181,11 +178,13 @@ Docker Compose is under active development. If you need help, would like to
 contribute, or simply want to talk about the project with like-minded
 individuals, we have a number of open channels for communication.
 
-* To report bugs or file feature requests: please use the [issue tracker on Github](https://github.com/docker/compose/issues).
+* To report bugs or file feature requests: please use the [issue tracker on
+* Github](https://github.com/docker/compose/issues).
```
Comment (shin-) [inline docs/overview.md]: This won't parse as a link because of the line break :(

File: docs/overview.md [hunk 2]
```diff
@@ -181,11 +178,13 @@ Docker Compose is under active development. If you need help, would like to
...
 
 * To talk about the project with people in real time: please join the
   `#docker-compose` channel on freenode IRC.
 
-* To contribute code or documentation changes: please submit a [pull request on Github](https://github.com/docker/compose/pulls).
+* To contribute code or documentation changes: please submit a [pull request on
+* Github](https://github.com/docker/compose/pulls).
```
Comment (shin-) [inline docs/overview.md]: Same as above

File: docs/rails.md [hunk 3]
```diff
@@ -11,7 +11,9 @@ weight=5
 
 ## Quickstart: Compose and Rails
 
-This Quickstart guide will show you how to use Compose to set up and run a Rails/PostgreSQL app. Before starting, you'll need to have [Compose installed](install.md).
+This Quickstart guide will show you how to use Compose to set up and run a
+Rails/PostgreSQL app. Before starting, you'll need to have [Compose
+installed](install.md).
```
Comment (shin-) [inline docs/rails.md]: This won't parse as a link because of the line break :(

File: docs/rails.md [hunk 4]
```diff
@@ -30,7 +32,12 @@ Dockerfile consists of:
     RUN bundle install
     ADD . /myapp
 
-That'll put your application code inside an image that will build a container with Ruby, Bundler and all your dependencies inside it. For more information on how to write Dockerfiles, see the [Docker user guide](https://docs.docker.com/engine/userguide/dockerimages/#building-an-image-from-a-dockerfile) and the [Dockerfile reference](https://docs.docker.com/engine/reference/builder/).
+That'll put your application code inside an image that will build a container
+with Ruby, Bundler and all your dependencies inside it. For more information on
+how to write Dockerfiles, see the [Docker user
+guide](https://docs.docker.com/engine/userguide/dockerimages/#building-an-image-from-a-dockerfile)
+and the [Dockerfile
+reference](https://docs.docker.com/engine/reference/builder/).
```
Comment (shin-) [inline docs/rails.md]: Same for those two here

**Comment by shin-** [comment]
Minor markdown quirks --

**Comment by londoncalling** [comment]
@shin- sorry, in trying to fix line wraps, I was overambitious. Should be good now after build, I think.

**Comment by shin-** [comment]
No worries, I know the feeling :) LGTM!

**Pull request merged**